assets/src/blocks/TableOfContents/TableOfContentsEditor.js

```diff
@@ -96,7 +96,7 @@ const renderView = (attributes, setAttributes, className) => {
   const {
     title,
     levels,
-    submenu_style,
+    table_of_contents_style,
```

Comment:
Same comment as earlier, should remain `submenu_style`

Reply:
Change reverted.